src/content/en/2021/structured-data.md

```diff
@@ -17,4 +19,568 @@ featured_stat_3: TODO
+
+In addition to this, social media platforms rely on structured data to influence how they read and display content when it's shared (or linked to) on their platforms. Rich previews, tailored titles and descriptions, and interactivity in these platforms are often powered by structured data.
+
+But there's more to see and understand here than search engine optimization and social media benefits. The scale, variety, impact and _potential_ of structured data goes far beyond rich results, far beyond search engines, and far beyond schema.org.
+
+For example, structured data facilitates:
+
+* Easier topic modelling and clustering across multiple pages, websites and concepts; enabling new types of research, comparison and services.
+* Enriching analytics data, to allow for deeper and horizontalized analysis of content and performance.
+* Creating a unified (or at least, connected) language and syntax for querying business systems and website content.
+* Semantic search; using the same rich metadata used for search engine optimization, to create and manage internal search systems.
+
+Whilst the findings of our research are inevitably shaped by the influence of search engines, we hope to also explore other types, formats, and use-cases of structured data.
+
+## Types of structured data and coverage
+
+Structured data comes in many formats, standards, and syntaxes. We've collected data about the most common of these across [our data set](./methodology).
+
+Specifically, we've identified and extracted structured data relating to:
+
+* [Schema.org](http://schema.org/)
+* <a hreflang="en" href="https://www.dublincore.org/specifications/dublin-core/">Dublin core</a>
+* Meta tags used by social networks:
+    * <a hreflang="en" href="https://ogp.me/">Open Graph</a>
+    * <a hreflang="en" href="https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started">Twitter</a>
+    * <a hreflang="en" href="https://developers.facebook.com/docs/sharing/webmasters/">Facebook</a>
+* [Microformats](http://microformats.org/) (and <a hreflang="en" href="https://microformats.org/wiki/microformats2">microformats2</a>)
+* [RDFa](https://en.wikipedia.org/wiki/RDFa), [Microdata](https://en.wikipedia.org/wiki/Microdata_(HTML)) and <a hreflang="en" href="https://json-ld.org/">JSON-LD</a>
+
+Collectively, these provide a broad overview of different use-cases and scenarios; and include both legacy standards and modern approaches (e.g., microformats vs JSON-LD).
+
+Before we explore specific usage across the various structured data types, we should briefly explore some caveats.
+
+### Data caveats
+
+#### 1. The influence of Content Management Systems
```

Comment:
Headings should have at least one sentence and not be empty (so called "Stacked headings", which are used to group headings but don't have any content themselves). "A heading is like a chapter title; it must have at least a sentence of information below it. Stacked headings can indicate inefficient organization of information."